packages/tests/custom-elements/html/shim.test.html

```diff
@@ -10,214 +10,109 @@
 -->
 <title>Custom Elements Tests</title>
 <meta charset="utf-8" />
+<script src="../../node_modules/@webcomponents/webcomponentsjs/bundles/webcomponents-pf_js.js"></script>
```

Comment:
Seems like this was necessary. The custom elements polyfill depends on `Promise` support (so every other test file already includes it) but this one originally didn't because it's for the native shim. The test environment now also depends on promises.

Reply:
I think it would be more proper for the test environment to include a promise polyfill itself and for this one import to be removed, but I'm getting a different set of errors in IE 11 with core-js' promise polyfill added. I'll keep trying for a little bit - maybe I've chosen an entry point that assumes other features that aren't available in IE 11.